xlive/Blam/Engine/structures/structure_bsp_definitions.cpp

```diff
@@ -0,0 +1,13 @@
+#include "stdafx.h"
+#include "structure_bsp_definitions.h"
+#include "Util/Memory.h"
+
+int16 get_global_structure_bsp_index(void)
```

Comment:
Move this back

Reply:
causes circular dependency issues.

Reply:
The data structures shouldn't go in the definitions headers to be honest,
those were really meant for their actual tagdefs which were separate.
Should probably move the structures from there to structures.h

Reply:
something we should work towards but doesn't solve the cyclical dependency issue. It'd just make it even wose.